monitor/widgets/src/main/java/org/datacleaner/monitor/scheduling/widgets/SchedulePanel.java

```diff
@@ -124,7 +137,8 @@ public SchedulePanel(final ClientConfig clientConfig, final ScheduleDefinition s
                 handler.showExecutionPopup();
             }
         }
-
+        
+       
```

Comment:
When I reformat the code these two lines are removed and replaced with the old empty line without white spaces.